### PR TITLE
Add Joel to web team page

### PR DESCRIPTION
**Pull request by tjkandala**
I don't know if this is an accurate description, I just copied Pooja's description from the search team. The main goal of this PR is to make the handbook reflect that Joel has started now.

### CODE REVIEW

File: handbook/engineering/web/index.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Before web team syncs, teammates and stakeholders should write down under "Discu
 
 ## Members
 
-- [J.K.](../../../company/team/index.md#todo)([Product Manager](../../product/roles/product_manager.md)) is starting on 2020-09-14. [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is involved in the meantime.
+- [Joel Kwartler](../../../company/team/index.md#todo) ([Product Manager](../../product/roles/product_manager.md)) is focused on web, [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is supporting.
```
Comment (christinaforney) [inline handbook/engineering/web/index.md:106]:
```suggestion
- [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him) ([Product Manager](../../product/roles/product_manager.md)) is focused on code insights, [Christina Forney](../../../company/team/index.md#christina-forney-she-her) is supporting.
```

**Review by christinaforney — APPROVED** [review]
Thanks @tjkandala! @Joelkw - we can merge this and then you can follow up with clarifications about what you will/won't be owning per our discussion yesterday.

**Review by Joelkw — APPROVED** [review]

**Comment by Joelkw** [comment]
@tjkandala @christinaforney that sounds good! I'll want to tweak this sometime next week, but want to still chat with Felix and potentially mention it in our sync so that everyone's given input on what would be most helpful for me to balance once I start focusing on code insights.

**Comment by christinaforney** [comment]
@Joelkw - sounds good. I'll merge this as is for now and you can update!

**Pull request merged**